packages/core/src/com/communication.ts

```diff
@@ -941,6 +938,7 @@ export class Communication {
                 this.pendingCallbacks.delete(callbackId);
                 delete this.callbacks[callbackId];
                 clearTimeout(timerId);
+                error.stack += `\nCaused by: ${JSON.stringify(cleanMessageForLog(message), null, 2)}`;
```

Comment:
wouldn't this affect source maps support? why add it to the stack (and trigger stack generation) and not to the .message?

Reply:
It can be in the message. trigger stack generation ? what do you mean? source map support still works.